src/rpc_client/src/meta_client.rs

```diff
@@ -1443,7 +1441,8 @@ macro_rules! for_all_meta_rpc {
         $macro! {
              { cluster_client, add_worker_node, AddWorkerNodeRequest, AddWorkerNodeResponse }
             ,{ cluster_client, activate_worker_node, ActivateWorkerNodeRequest, ActivateWorkerNodeResponse }
-            ,{ cluster_client, delete_worker_node, DeleteWorkerNodeRequest, DeleteWorkerNodeResponse }
+            //(not used) ,{ cluster_client, delete_worker_node, DeleteWorkerNodeRequest, DeleteWorkerNodeResponse }
```

Comment:
We still need to delete the worker.

Reply:
True, we will need this function, but we are currently not using it, which is why `risedev c` fails. Cloud is currently working on the deletion process. In the future the `delete_worker_node` rcp call will not delete the node, but instead only tell meta that this node will be deleted. The deletion of the K8s pod itself will be done by a cloud workflow. 

CC @ALeitert.

Reply:
I added a simple client function which uses `delete_worker_node` to the `meta_client`, so we get rid of the warning.